internal/cli/atlas/data_lake_describe_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package atlas
+
+import (
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/mongodb/mongocli/internal/mocks"
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+func TestDataLakeDescribe_Run(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	mockStore := mocks.NewMockDataLakeDescriber(ctrl)
+
+	defer ctrl.Finish()
+
+	expected := mongodbatlas.DataLake{
```

Comment:
FYI, you could just do `mongodbatlas.DataLake{}` since you don't depend on any of the values being returned

Reply:
(will add this on the next PR)